components/script/dom/document.rs

```diff
@@ -450,6 +452,91 @@ impl<'a> DocumentHelpers<'a> for JSRef<'a, Document> {
             None => {}
         }
     }
+
+    fn handle_mouse_move_event (self, js_runtime: *mut JSRuntime, point: Point2D<f32>, mouse_over_targets: &mut Option<Vec<JS<Node>>>) -> bool{
```

Comment:
Also can you wrap this line, and the line added in the `trait`, to fit within 100 columns (ideally, 80)?